init master

hanseongseo opened the pull request.

kojoo112 committed:
asdf

kojoo112 committed:
Merge pull request #118 from kojoo112/feature/asef

asdf

kojoo112 committed:
rename timetabledto

kojoo112 committed:
Merge pull request #119 from kojoo112/feature/rename-timetabledto

rename timetabledto

hanseongseo committed:
scheduler enable

hanseongseo committed:
Merge pull request #120 from hanseongseo/feature/admin-reservation-timetable

scheduler enable

hanseongseo committed:
가예약 자동 취소 기능 구현

hanseongseo committed:
Merge pull request #121 from hanseongseo/develop

가예약 자동 취소 기능 구현

kojoo112 committed:
가격 관리 수정

kojoo112 committed:
Merge pull request #122 from kojoo112/feature/refactoring-admin-price

가격 관리 수정

hanseongseo committed:
스케줄러(빈예약 생성) 기능 수정
서버 타임존 추가

hanseongseo committed:
Merge pull request #123 from hanseongseo/feature/fix-reservation-batch

스케줄러(빈예약 생성) 기능 수정

kojoo112 committed:
reservation, reservation-history column 수정

kojoo112 committed:
Merge pull request #124 from kojoo112/feature/modify-reservation-history-response

reservation, reservation-history column 수정

hanseongseo committed:
reservation roomType 제거에 따른 admin 예약 목록 페이지 수정

hanseongseo committed:
api - getThemesWithReservations(예약 페이지용 지점별 예약현황 조회) 쿼리 최적화

hanseongseo committed:
Merge pull request #125 from hanseongseo/feature/fix-admin-reservation-view

reservation roomType 제거에 따른 admin 예약 목록 페이지 수정

hanseongseo committed:
Merge pull request #126 from hanseongseo/feature/refactoring-jpa-query

api - getThemesWithReservations(예약 페이지용 지점별 예약현황 조회) 쿼리 최적화

kojoo112 committed:
send cancel template

kojoo112 committed:
Merge pull request #127 from kojoo112/feature/send-cancel-template

send cancel template

kojoo112 committed:
cors config 수정

kojoo112 committed:
Merge pull request #128 from kojoo112/feature/cors-config

cors config 수정

hanseongseo committed:
어드민 예약목록 배경 수정, queryparam 날짜 optional 변경(default: 현재날짜)

hanseongseo committed:
Merge pull request #129 from hanseongseo/feature/admin-fix

어드민 예약목록 배경 수정, queryparam 날짜 optional 변경(default: 현재날짜)

hanseongseo committed:
가예약 형식 수정

kojoo112 committed:
return api exception

kojoo112 committed:
Merge pull request #130 from kojoo112/feature/return-api-exception

return api exception

kojoo112 committed:
cors config 오타 수정

kojoo112 committed:
Merge pull request #131 from kojoo112/feature/cors-config

cors config 오타 수정

hanseongseo committed:
get - /reservations 로그 추가

hanseongseo committed:
Merge pull request #132 from hanseongseo/feature/admin-fix

get - /reservations 로그 추가

hanseongseo committed:
개발 db url 적용(active profile local)

hanseongseo committed:
Merge pull request #133 from hanseongseo/feature/admin-fake-reservation-timer

개발 db url 적용(active profile local)

rkb94 committed:
modify get reservation list api

rkb94 committed:
Merge pull request #134 from rkb94/modify-get-reservation-list-api

modify get reservation list api

rkb94 committed:
add-time-zone-to-api

rkb94 committed:
Merge pull request #135 from rkb94/add-time-zone-to-api

add-time-zone-to-api

hanseongseo committed:
어드민 예약목록 타이머 기능 구현 중 - 저장용 커밋

hanseongseo committed:
어드민 예약목록 가예약 타이머 기능 완료

kojoo112 committed:
price timetable 수정 구현

kojoo112 committed:
Merge pull request #136 from kojoo112/feature/modify-price-timetable

price timetable 수정 구현

hanseongseo committed:
admin hikari CP 설정
      max-lifetime: 58000
      maximum-pool-size: 25

hanseongseo committed:
scheduler 활성화

hanseongseo committed:
Merge pull request #137 from hanseongseo/feature/connection-pool-config

어드민 예약목록 타이머 기능 및 connection pool sizing

kojoo112 committed:
가예약 관리 init

hanseongseo committed:
admin security config

hanseongseo committed:
Account 엔티티, dto

hanseongseo committed:
db 수정 sql

kojoo112 committed:
create mock reservations 백업

kojoo112 committed:
save mock reservation

kojoo112 committed:
Merge pull request #138 from kojoo112/feature/mock-reservations

Feature/mock reservations

hanseongseo committed:
Merge branch 'chadev94:develop' into feature/admin-security

hanseongseo committed:
admin reservation 예약목록 css 수정 및 지점 이동 버튼 추가(dropdown 제거)

hanseongseo committed:
Merge pull request #139 from hanseongseo/feature/admin-reservation-reactive-css

admin reservation 예약목록 css 수정 및 지점 이동 버튼 추가(dropdown 제거)

hanseongseo committed:
Merge branch 'chadev94:develop' into feature/admin-security

kojoo112 committed:
logo width 조정

kojoo112 committed:
Merge pull request #140 from kojoo112/feature/add-width-logo

logo width 조정

kojoo112 committed:
make reservation interface

kojoo112 committed:
Merge pull request #141 from kojoo112/feature/make-reservation-interface

make reservation interface

hanseongseo committed:
admin 회원 등록 구현

hanseongseo committed:
Merge branch 'chadev94:develop' into feature/admin-security

kojoo112 committed:
취소 알림 에러수정

kojoo112 committed:
Merge pull request #142 from kojoo112/feature/fix-cancel-notify

취소 알림 에러수정

kojoo112 committed:
일괄예약 등록시 예약돼있는 예약 제외후 가예약

kojoo112 committed:
Merge pull request #143 from kojoo112/feature/modify-mock-reservations-logic

일괄예약 등록시 예약돼있는 예약 제외후 가예약

hanseongseo committed:
Merge branch 'chadev94:develop' into feature/admin-security

hanseongseo committed:
admin 계정 구현(예약목록은 UI 수정 후 적용)

hanseongseo committed:
Merge pull request #144 from hanseongseo/feature/admin-security

admin 계정 기능 구현(예약 목록은 UI 수정 후 구현)

hanseongseo committed:
Revert "admin 계정 기능 구현(예약 목록은 UI 수정 후 구현)"

hanseongseo committed:
Merge pull request #145 from chadev94/revert-144-feature/admin-security

Revert "admin 계정 기능 구현(예약 목록은 UI 수정 후 구현)"

hanseongseo committed:
Hot fix, 중복 예약 시 예약 수정되는 버그 수정

hanseongseo committed:
Merge pull request #146 from hanseongseo/develop

Hot fix, 중복 예약 시 예약 수정되는 버그 수정

kojoo112 committed:
reservation logic 수정

rkb94 committed:
Add validation when mock reservation action

rkb94 committed:
Merge pull request #147 from rkb94/feature/add-validation-when-mock-reservation

Add validation when mock reservation action

kojoo112 committed:
registerMockReservation 수정

kojoo112 committed:
reservationDto date 응답값 수정

kojoo112 committed:
Merge pull request #148 from kojoo112/feature/modify-reservation-logic

Feature/modify reservation logic

kojoo112 committed:
open room logic 수정

kojoo112 committed:
Merge pull request #149 from kojoo112/feature/modify-open-room-logic

open room logic 수정

rkb94 committed:
Change getReservedBy of ReservationDto

rkb94 committed:
Merge pull request #150 from rkb94/openroom-admin

Change getReservedBy of ReservationDto

kojoo112 committed:
openRoom logic 추가

kojoo112 committed:
Merge pull request #151 from kojoo112/feature/add-open-room-logic

openRoom logic 추가

kojoo112 committed:
cancel 버그 수정

kojoo112 committed:
Merge pull request #152 from kojoo112/feature/fix-cancel-reservation

cancel 버그 수정

kojoo112 committed:
/reservation-histories registeredAt 추가

kojoo112 committed:
Merge pull request #153 from kojoo112/feature/add-reservation-registered-at

/reservation-histories registeredAt 추가

kojoo112 committed:
roomType null check

kojoo112 committed:
Merge pull request #154 from kojoo112/feature/room-type-null-check

roomType null check

hanseongseo committed:
admin security 적용
TODO: AdminViewController reservation 페이지 security 적용

hanseongseo committed:
Merge branch 'chadev94:develop' into feature/admin-security

hanseongseo committed:
admin 예약목록 페이지 security 적용

hanseongseo committed:
admin 예약목록 페이지 datepicker value 할당 방식 변경

hanseongseo committed:
Merge pull request #155 from hanseongseo/feature/admin-security

admin security 적용

hanseongseo committed:
Revert "admin security 적용"

hanseongseo committed:
Merge pull request #156 from chadev94/revert-155-feature/admin-security

Revert "admin security 적용"

hanseongseo committed:
reservation batch scheduler 비활성화

hanseongseo committed:
reservation batch api - 프로시저 호출 방식으로 변경

hanseongseo committed:
Merge pull request #157 from hanseongseo/feature/change-batch-to-procedure

reservation batch 프로시저로 변경 및 프로시저 호출 api 구현

kojoo112 committed:
reservation.js 내 에러코드 삭제(햣)

kojoo112 committed:
Merge pull request #158 from kojoo112/hotfix/fix-error-in-reservation-js

reservation.js 내 에러코드 삭제(햣)

hanseongseo committed:
admin security 적용

hanseongseo committed:
Merge branch 'develop' into feature/admin-security#2

hanseongseo committed:
Merge pull request #159 from hanseongseo/feature/admin-security#2

admin security 적용

mhj94 committed:
Add RootFilter and ServletConfig

rkb94 committed:
Merge pull request #160 from mhj94/add-root-filter-to-admin

Add RootFilter and ServletConfig

mhj94 committed:
Merge pull request #1 from chadev94/develop

Update develop by upstream

hanseongseo committed:
remember-me, custom login page 적용

hanseongseo committed:
Merge branch 'develop' into feature/remember-me

hanseongseo committed:
Merge pull request #161 from hanseongseo/feature/remember-me

remember-me, custom login page 적용

mhj94 committed:
reservation.html 호출시 쿼리 갯수 줄임

mhj94 committed:
Merge pull request #162 from mhj94/feature/reduce-query-in-reservation-view

reservation.html 호출시 쿼리 갯수 줄임

mhj94 committed:
Add command

mhj94 committed:
Add command 2

mhj94 committed:
Merge pull request #163 from mhj94/feature/test-branch

Feature/test branch

mhj94 committed:
refactoring-reservation

mhj94 committed:
Merge pull request #164 from mhj94/feature/refactoring-reservations

refactoring-reservation

hanseongseo committed:
theme 도메인 runningTime 추가

mhj94 committed:
reminder

kojoo112 committed:
CorsConfig coreModule -> apiModule 이동

kojoo112 committed:
Merge pull request #167 from kojoo112/feature/move-cors-config-to-api-module

CorsConfig coreModule -> apiModule 이동

kojoo112 committed:
각각의 모듈에 service구현, 코어에만 repository 생성

kojoo112 committed:
Merge pull request #168 from kojoo112/feature/refactoring-service-repository-class

각각의 모듈에 service구현, 코어에만 repository 생성

mhj94 committed:
리마인더 service로직 수정

kojoo112 committed:
core service 삭제

kojoo112 committed:
Merge pull request #169 from kojoo112/feature/remove-core-service

core service 삭제

kojoo112 committed:
인메모리 redis로 카카오톡 API 카운트

hanseongseo committed:
timer 중간커밋

hanseongseo committed:
Merge branch 'chadev94:develop' into feature/admin-timer

mhj94 committed:
리마인더 조회 쿼리 수정 및 스케쥴러 시간 설정

mhj94 committed:
Merge pull request #166 from mhj94/feature/reminder

reminder

mhj94 committed:
리마인더 스케쥴러 초단위에서 분단위로 변경

rkb94 committed:
Merge pull request #171 from mhj94/feature/reminder

리마인더 스케쥴러 초단위에서 분단위로 변경

mhj94 committed:
리마인더 스케쥴러 분단위 재수정

rkb94 committed:
Merge pull request #172 from mhj94/feature/reminder

리마인더 스케쥴러 분단위 재수정

hanseongseo committed:
admin timer, toast popup

kojoo112 committed:
Merge remote-tracking branch 'upstream/develop' into feature/make-kakao-talk-api-count-method

hanseongseo committed:
Merge branch 'chadev94:develop' into feature/admin-timer

hanseongseo committed:
admin timer - 시간 초과 시 text 색상 변경

hanseongseo committed:
Merge pull request #174 from hanseongseo/feature/admin-timer

admin timer

mhj94 committed:
hint 엔티티 생성

rkb94 reviewed (approved):
ㅋㅋ is thos only develop code??

kojoo112 committed:
checkReservationCount 로직 변경

kojoo112 committed:
checkReservationCount 로직 변경

kojoo112 committed:
메서드명 변경

mhj94 committed:
hint 엔티티 hint_id 컬럼 수정

kojoo112 committed:
Merge pull request #176 from mhj94/feature/make-hintEntity

hint 엔티티 생성

kojoo112 committed:
Merge remote-tracking branch 'upstream/develop' into feature/make-kakao-talk-api-count-method

kojoo112 committed:
Merge pull request #170 from kojoo112/feature/make-kakao-talk-api-count-method

인메모리 redis로 카카오톡 API 카운트

kojoo112 committed:
Revert "Merge pull request #170 from kojoo112/feature/make-kakao-talk-api-count-method"

This reverts commit 8ca11060d6b5126907c3d765a22b0693e8d391c9, reversing
changes made to b3b0617e1db2826fdddc95c2cfc019272def0f39.

kojoo112 committed:
Revert "인메모리 redis로 카카오톡 API 카운트"

kojoo112 committed:
Merge pull request #177 from chadev94/revert-170-feature/make-kakao-talk-api-count-method

Revert "인메모리 redis로 카카오톡 API 카운트"

kojoo112 committed:
Merge remote-tracking branch 'upstream/develop' into develop

kojoo112 committed:
exception logic 변경, application @EnableJpaRepositories 변경

kojoo112 committed:
Merge pull request #178 from kojoo112/feature/modify-exception-logic

exception logic 변경, application @EnableJpaRepositories 변경

mhj94 committed:
Add ThemeDto,ThemeService,DtoConverter logic and implement HintDto,HintRepository

mhj94 committed:
Revert "Add ThemeDto,ThemeService,DtoConverter logic and implement HintDto,HintRepository"

This reverts commit fc2bf50238380e8b2e99171fd1fc00f6b0dee21a.

mhj94 committed:
Revert "Revert "Add ThemeDto,ThemeService,DtoConverter logic and implement HintDto,HintRepository""

This reverts commit 458bf0803c14777cbc7e55cc8e6803f20cceecd7.

mhj94 committed:
Add ThemeService,HintService logic and Fix Hint,HintDto and Remove DtoConverter logic, Update ApiRestController

kojoo112 committed:
Merge remote-tracking branch 'upstream/develop' into develop

kojoo112 committed:
runningTime 수정 추가

kojoo112 committed:
Merge pull request #180 from kojoo112/feature/add-running-time-to-theme-setting

Feature/add running time to theme setting

kojoo112 committed:
불필요 데이터 삭제

kojoo112 committed:
Merge pull request #181 from kojoo112/feature/remove-unnecessary-file

불필요 데이터 삭제

hanseongseo committed:
admin timer - 버그 개선

- 숫자 00:00 포맷
- 시간 초과 시 시간계산 맞지 않는 것 개선
- 토스트 팝업 자동꺼짐 비활성화
- 토스트 팝업 기능 개선, 종료/시간초과 알림 추가
- 타이머 진행 중 text color 변경
- setInterval로 인해 발생하는 시간 텀 동안 진행 중인 테마는 로딩 스피너 돌게 구현

hanseongseo committed:
Merge pull request #182 from hanseongseo/admin-timer-debug

admin timer - 버그 개선

- 숫자 00:00 포맷
- 시간 초과 시 시간계산 맞지 않는 것 개선
- 토스트 팝업 자동꺼짐 비활성화
- 토스트 팝업 기능 개선, 종료/시간초과 알림 추가
- 타이머 진행 중 text color 변경
- setInterval로 인해 발생하는 시간 텀 동안 진행 중인 테마는 로딩 스피너 돌게 구현

kojoo112 committed:
Merge pull request #179 from mhj94/feature/make-hintEntity

Feature/make hint entity

rkb94 committed:
Upgrade reservation page

rkb94 committed:
Update get reservation list api

rkb94 committed:
Add runningTime to api

rkb94 committed:
Update UI of timer alert

hanseongseo committed:
json 업로드 기능

hanseongseo committed:
Merge pull request #183 from hanseongseo/feature/publish-theme-json

json 업로드 기능

hanseongseo committed:
json 업로드 프론트 기능
- TODO: 업로드 후 처리(알림 등)

hanseongseo committed:
Merge pull request #184 from hanseongseo/feature/admin-json-publish

json 업로드 프론트 기능

hanseongseo committed:
theme 생성 UI

hanseongseo committed:
json path 설정 추가

hanseongseo committed:
Merge pull request #185 from hanseongseo/feature/admin-create-theme

theme 생성 UI

hanseongseo committed:
테마 생성

hanseongseo committed:
Merge branch 'chadev94:develop' into feature/admin-create-theme

hanseongseo committed:
테마 생성 UI

hanseongseo committed:
테마 등록 기능

- s3 버킷 분리(xcape-business-sdk-uploads-dev)

hanseongseo committed:
Merge pull request #186 from hanseongseo/feature/admin-create-theme

테마 등록 기능

hanseongseo committed:
배너 json 발행
- 테마 생성 오류 수정

hanseongseo committed:
Merge pull request #187 from hanseongseo/feature/banner-json-publish

배너 json 발행

mhj94 committed:
Merge remote-tracking branch 'upstream/develop' into develop

kojoo112 committed:
가맹점 등록 폼 생성 및 db not null 처리

kojoo112 committed:
Merge pull request #188 from kojoo112/feature/chadev-001

가맹점 등록 폼 생성 및 db not null 처리

kojoo112 committed:
로그인 후 css 파일로 가는 버그 수정

kojoo112 committed:
Merge pull request #189 from kojoo112/hotfix/redirect-to-reservations

로그인 후 css 파일로 가는 버그 수정

mhj94 committed:
Merge remote-tracking branch 'upstream/develop' into develop

kojoo112 committed:
delete businessIcon

kojoo112 committed:
Merge pull request #190 from kojoo112/feature/delete-business-icon

delete businessIcon

mhj94 committed:
Merge remote-tracking branch 'upstream/develop' into feature/chadev-004

mhj94 committed:
Merge remote-tracking branch 'upstream/develop' into develop

mhj94 committed:
Feat : 어빌리티 저장 추가

mhj94 committed:
Fix : 테마 생성 시 어빌리티 저장 오류 수정

mhj94 committed:
Merge pull request #191 from mhj94/feature/chadev-004

Feature/chadev 004

mhj94 committed:
Merge remote-tracking branch 'upstream/develop' into develop

hanseongseo committed:
azure sql 스케줄러 기능 미지원으로 예약 배치 admin was 적용
- TODO: 임시 비활성화, azure sql 마이그레이션 후 활성화

hanseongseo committed:
Merge pull request #192 from hanseongseo/feature/reservation-batch

azure sql 스케줄러 기능 미지원으로 예약 배치 admin was 적용

kojoo112 committed:
오픈룸 예약 취소시 price NPE 디버깅

kojoo112 committed:
Merge pull request #193 from kojoo112/feature/chadev-002

오픈룸 예약 취소시 price NPE 디버깅

kojoo112 committed:
예약 취소시 룸타입 npe fix

kojoo112 committed:
Merge pull request #194 from kojoo112/hotfix/room-type-npe

예약 취소시 룸타입 npe fix

mhj94 committed:
Merge remote-tracking branch 'upstream/develop' into develop

mhj94 committed:
Feat : 가맹점 수정 등록 페이지 및 기능 추가

mhj94 committed:
Feat : 호버 기능 추가

mhj94 committed:
Fix : 가맹점 만들기 버튼 이동

mhj94 committed:
Feat : 호버 클릭시 수정폼 추가

mhj94 committed:
Fix : 가맹점 수정폼 수정

mhj94 committed:
Feat : 가맹점 수정폼 수정

mhj94 committed:
Fix : 가맹점 등록 및 수정 기능 수정

mhj94 committed:
Fix : 가맹점 수정 실패시, 폼 초기화 되도록 수정

mhj94 committed:
Style : 변수명 변경

mhj94 committed:
Fix : DOMContentLoaded 삭제

mhj94 committed:
Fix : 가맹점 목록 페이지 수정

mhj94 committed:
Feat : 가맹점 오름차순 조회 추가

mhj94 committed:
Feat : 테마 조회 기능 및 발행 기능 추가

mhj94 committed:
Feat : 가맹점 발행 기능 추가

mhj94 committed:
Style : 공백 수정

mhj94 committed:
Style : 폴더명 변경

mhj94 committed:
Style : 메서드명 변경

mhj94 committed:
Merge pull request #196 from mhj94/feature/chadev-005

Feature/chadev 005

hanseongseo committed:
admin 개발서버 cicd 구성

hanseongseo committed:
admin 개발서버 cicd

hanseongseo committed:
admin 개발서버 cicd 경로수정

hanseongseo committed:
개발서버 admin cicd 스크립트

hanseongseo committed:
github actions 버전 변경

hanseongseo committed:
gitignore gradle 비활성화

hanseongseo committed:
Merge pull request #1 from hanseongseo/feature/admin-cicd

Feature/admin cicd

hanseongseo committed:
중복 워크플로우 삭제

hanseongseo committed:
빌드 파일 경로 수정

hanseongseo committed:
빌드 파일 업로드 경로 수정

hanseongseo committed:
s3 환경변수 변경 반영

hanseongseo committed:
빌드 파일 경로 수정

hanseongseo committed:
Update dev-admin.yml

hanseongseo committed:
Update dev-admin.yml

hanseongseo committed:
Update dev-admin.yml

hanseongseo committed:
Update dev-admin.yml

hanseongseo committed:
ec2 접속 테스트

hanseongseo committed:
ec2 접속 테스트

hanseongseo committed:
Update dev-admin.yml

hanseongseo committed:
Merge pull request #2 from hanseongseo/feature/admin-cicd

Feature/admin cicd

hanseongseo committed:
Update dev-admin.yml

hanseongseo committed:
Merge pull request #197 from hanseongseo/develop

개발서버 admin cicd

hanseongseo committed:
github actions 설정 제거

hanseongseo committed:
Merge pull request #198 from hanseongseo/feature/remove-git-actions

github actions 설정 제거

hanseongseo committed:
datasource ec2 개발서버로 변경

hanseongseo committed:
Merge pull request #199 from hanseongseo/feature/remove-git-actions

datasource ec2 개발서버로 변경
